core/tests/revert-test/tests/revert-and-restart-en.test.ts

```diff
@@ -359,10 +398,11 @@ describe('Block reverting test', function () {
         mainNode = await MainNode.spawn(mainLogs, enableConsensus, true);
 
         console.log('Wait for the external node to detect reorg and terminate');
-        await extNode.waitForExit();
+        await utils.sleep(3);
```

Comment:
could we keep some other checks, not sleeps, most of the time it doesn't work properly

Reply:
For the reason above, we can restore also this. Addressed in https://github.com/matter-labs/zksync-era/pull/2408/commits/06d0f97a464b697d052be4745162258638086f92